pom.xml

```diff
@@ -8,10 +8,20 @@
 
 	 <properties>
         <bc.version>1.68</bc.version>
+        <osgi.imported.packages>
+            !junit,
+            !junit.*,
+            !com.apple.*,
+            !com.lowagie.toolbox,
+            !com.lowagie.text.*,
+            org.bouncycastle.*;version="[1.63,2)",
```

Comment:
Does it actually work with 1.63 or only with 1.68?

Reply:
Our source changes were made to compile with 1.68. I found however that the earliest version our changes could compile with was 1.63. If I have no version string here, the import directive has version 1.68. I decided to have a range here just in case.